modules/nw-sriov-example-dpdk-line-rate.adoc

```diff
@@ -0,0 +1,278 @@
+
+The following is an example of `sriovNetworkNodePolicy` for Mellanox NICs:
+[source,yaml]
+----
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetworkNodePolicy
+    metadata:
+     name: dpdk-nic-1
+     namespace: openshift-sriov-network-operator
+    spec:
+     deviceType: netdevice <1>
+     isRdma: true <2>
+     nicSelector:
+       rootDevices:
+         - "0000:5e:00.1"
+     nodeSelector:
+       node-role.kubernetes.io/worker-cnf: ""
+     numVfs: 5
+     priority: 99
+     resourceName: dpdk_nic_1
+    ---
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetworkNodePolicy
+    metadata:
+     name: dpdk-nic-2
+     namespace: openshift-sriov-network-operator
+    spec:
+     deviceType: netdevice
+     isRdma: true
+     nicSelector:
+       rootDevices:
+         - "0000:5e:00.0"
+     nodeSelector:
+       node-role.kubernetes.io/worker-cnf: ""
+     numVfs: 5
+     priority: 99
+     resourceName: dpdk_nic_2
+----
+<1> For Mellanox devices the `deviceType` must be `netdevice`.
+<2> For Mellanox devices `isRDMA` must be `true`.
+Mellanox cards are connected to DPDK applications using Flow Bifurcation. This mechanism splits traffic between Linux user space and kernel space, and can enhance line rate processing capability.
+
+== Using the SR-IOV network operator to create the `sriovNetwork` object
+The following is an example definition of an `sriovNetwork` object. In this case Intel and Mellanox configurations are identical.
+[source,yaml]
+----
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetwork
+    metadata:
+     name: dpdk-network-1
+     namespace: openshift-sriov-network-operator
+    spec:
+     ipam: '{"type": "host-local","ranges": [[{"subnet": "10.0.1.0/24"}]],"dataDir":
+       "/run/my-orchestrator/container-ipam-state-1"}' <1>
+     networkNamespace: dpdk-test <2>
+     spoofChk: "off"
+     trust: "on"
+     resourceName: dpdk_nic_1 <3>
+    ---
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetwork
+    metadata:
+     name: dpdk-network-2
+     namespace: openshift-sriov-network-operator
+    spec:
+     ipam: '{"type": "host-local","ranges": [[{"subnet": "10.0.2.0/24"}]],"dataDir":
+       "/run/my-orchestrator/container-ipam-state-1"}'
+     networkNamespace: dpdk-test
+     spoofChk: "off"
+     trust: "on"
+     resourceName: dpdk_nic_2
+----
+<1> You can use a different IPAM, such as Whereabouts. See "Dynamic IP address assignment configuration with Whereabouts" for more information.
+<2> You must request the `networkNamespace` where the network attachment definition will be created. The `sriovNetwork` CR must be created under the `openshift-sriov-network-operator` namespace.
+<3> The `resourceName` value must match that of the `resourceName` created under the `sriovNetworkNodePolicy`.
+
+== Running a DPDK base workload
```

Comment:
Procedure modules should have only one heading. You could separate out this section to a separate module with the required metadata.

Also, in the new file you could add a content type label defining this either as a procedure or concept module. If it is a procedure module, please add the required steps and `.Procedure` dot heading.